CRM/Utils/Number.php

```diff
@@ -88,6 +88,7 @@ public static function createTruncatedDecimal($keyValue, $precision) {
   public static function formatUnitSize($size, $checkForPostMax = FALSE) {
     if ($size) {
       $last = strtolower($size{strlen($size) - 1});
+      $size = (int)$size;
```

Comment:
@Jazz-Man you need to have 1 space between the (int) and $size

Reply:
It's all through github's online editor.
Now everything is fine, testing goes on.